### PR TITLE
New user admin view

### PR DESCRIPTION
**Pull request by DannyRL**
Changes:
1) Made signup page only accessible to Admins
2) Added button on list-users view to add new user
3) After submitting new user, it redirects back to list-users page

### CODE REVIEW

**Comment by hxghhhh** [comment]
@ssk13 @amahfood Review Pleasee

**Comment by ssk13** [comment]
Currently, if he tries to add an invalid user, clicking the "create" button still redirects to users without adding anything to the db - probably should not redirect in that case

File: modules/users/client/views/authentication/signup.client.view.html [hunk 1]
```diff
@@ -63,9 +63,7 @@ <h3 class="col-md-12 text-center">Sign Up</h3>
           <progressbar value="strengthProgress" type="{{strengthColor}}"><span style="color:white; white-space:nowrap;">{{strengthProgress}}%</span></progressbar>
         </div>
         <div class="text-center form-group">
-          <button type="submit" class="btn btn-primary">Sign up</button>
-          &nbsp; or&nbsp;
-          <a ui-sref="authentication.signin" class="show-signup">Sign in</a>
+          <button onclick="location.href='/admin/users'" type="submit" class="btn btn-primary">Create</button>
```
Comment (ssk13) [inline modules/users/client/views/authentication/signup.client.view.html]: instead, probably make this call a method onclick that checks if the information is valid

**Comment by amahfood** [comment]
The code looks goo to me. I couldn't checkout the specific branch because of merge conflicts that I can't see because Atom is still buggy.

**Pull request merged**